.github/release-please-config.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+    "changelog-sections": [
+        {
+            "type": "feat",
+            "section": "Features",
+            "hidden": false
+        },
+        {
+            "type": "fix",
+            "section": "Bug Fixes",
+            "hidden": false
+        }
```

Comment:
If you want to include other types in your changelog like chore, refactor, or docs we can add those here.